src/System.Management/tests/System/Management/ManagementDateTimeConverterTests.cs

```diff
@@ -11,15 +11,16 @@ public class ManagementDateTimeConverterTests
         [ConditionalFact(typeof(WmiTestHelper), nameof(WmiTestHelper.IsWmiSupported))]
         public void DateTime_RoundTrip()
         {
-            var date = new DateTime(2002, 4, 8, 14, 18, 35, 978, DateTimeKind.Utc);
-            var dmtfDate = "20020408141835.978000+000";
+            var date = new DateTime(2002, 4, 8, 14, 18, 35, 978, DateTimeKind.Utc).AddMinutes(150);
+            var dmtfDate = "20020408141835.978000-150";
+            var dmtfDateExpected = "20020408164835.978000+000";
```

Comment:
>"20020408164835.978000+000"; [](start = 35, length = 28)

does this value will depend on the current local time zone on the machine? 

could you please try to set your TZ to UTC on your machine and find out if the test will pass with your changes?

thanks.

Reply:
No it does not depend on my time zone, I select it random. My time zone is UTC+7 but here I select (UTC-2:30). (UTC-2:30) may be does not exist in real.